requirements.txt

```diff
@@ -11,5 +11,5 @@ paho-mqtt>=1.5.1
 pandas_datapackage_reader>=0.18.0
 tables>=3.6.1
 python-Levenshtein>=0.12.2
-paho-mqtt
+paho-mqtt==1.5.1
```

Comment:
why not >= 1.5.1?

Reply:
Paho introduced breaking changes in version 1.6.0 making the code fail. As I did not find a good changelog, I decided that it should be ok to keep the older version, as it is is only used in the test case. 
If wanted it could be made working with the newer paho version, with a bit of effort